src/main/resources/avro/assaymetadata.avdl

```diff
@@ -0,0 +1,146 @@
+@namespace("org.ga4gh.models")
+
+/**
+This protocol defines metadata objects describing experimental or analytical procedures, performed on biological objects (e.g. biosamples) or readouts from experimental procedures.
+*/
+
+protocol Assaymetadata {
+
+  import idl "common.avdl";
+  import idl "metadata.avdl";
+
+  /**
+  An analysis contains an interpretation of one or several experiments.
+  (e.g. SNVs, copy number variations, methylation status) together with
+  information about the methodology used.
```

Comment:
wording subject to misinterpretation this is only the provenance, it does not contain the actual interperatation